### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: src/setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 setup(
     name='katello-agent',
```
Comment (ekohl) [inline src/setup.py:21]: This means it'll be published as katello-agent on PyPI. Should we rename this to katello-host-tools instead? That probably makes it a major version bump though.

File: src/setup.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
...
-    version='3.5.7',
+    version='3.5.8',
     description='Katello Agent',
     author='Jeff Ortel',
     author_email='jortel@redhat.com',
```
Comment (ekohl) [inline src/setup.py:25]: While we're at it, should this also be updated?

**Comment by ekohl** [comment]
And on a related note: https://github.com/Katello/katello-host-tools/pull/128

File: src/setup.py (hunk 2 above)
Comment (jturel) [reply src/setup.py:25]: Must author and author_email be specified? There is no singular author or point of contact, so if I can remove them safely then that'd be good

Comment (ekohl) [reply src/setup.py:25]: https://python-packaging.readthedocs.io/en/latest/minimal.html suggests it does. If you remove it, you get this:

> warning: check: missing meta-data: either (author and author_email) or (maintainer and maintainer_email) should be supplied

As the author you could say something like `The Foreman Project` but not sure what the email should be. Perhaps something generic as `no-email@theforeman.org`?

File: src/setup.py [hunk 3]
```diff
@@ -18,27 +18,22 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='katello-agent',
-    version='3.5.7',
-    description='Katello Agent',
-    author='Jeff Ortel',
-    author_email='jortel@redhat.com',
+    name='katello-host-tools',
+    version='3.5.8',
```
Comment (ekohl) [inline src/setup.py:22]: I think I'd make this at least 3.6.0. Renaming it may have impact on packaging as well.
```suggestion
    version='3.6.0',
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: src/setup.py (hunk 3 above)
Comment (jturel) [reply src/setup.py]: Bumped to 4.0 - seemed appropriate

**Review by ekohl — APPROVED** [review]
Looks good to me. Feel free to merge and tag.

**Pull request merged**